docs/docs/dev_guide/validation_plugins.md

```diff
@@ -251,6 +244,110 @@ validation:
 
 ***NOTE***: Without the `implementation_file` the initial validator plugin will spit out an error saying that an implementation file is missing. This is needed for the validator plugin to function properly.
 
+## Implementation of a Validation Plugin for Primitive Types
+
+Examples of a primitive validator plugin can be found by refering the first-party provider for primitive validations [primitive-type-check](https://github.com/jondavid-black/AaC/tree/main/python/src/aac/plugins/first_party/primitive_type_check)
+
+Looking inside this directory the structure of this plugin looks other AaC plugins:
```

Comment:
I'm not sure what exactly to put here because I don't know exactly what is supposed to be conveyed but "[...] plugin looks other [...]" doesn't seem right.

Reply:
I don't see the referenced issue in the comment. Can you elaborate if it hasn't been addressed?

Reply:
What I was trying to say is that this line reads strangely - specifically, the part about "the structure of this plugin looks other AaC plugins". Re-reading it, perhaps the intent is this:

```suggestion
Looking inside this directory, the structure of this plugin looks like other AaC plugins:
```